translation/models/translation_services.py

```diff
@@ -33,7 +33,7 @@ def create_text_translation(self, text, **kwargs):
         # if text is long html, &lang param isn't parsed properly
         params.append("&{0}={1}".format("text", text))
         response = requests.get(final_url + "".join(params)).json()
-        return response['text']
+        return " ".join(response['text'])
```

Comment:
Actually, `response['text']` is a list with single element, but for sure I use " " if we have 2 or more elements and don't want to mess them.